pkgs/development/python-modules/geopandas/default.nix

```diff
@@ -14,6 +14,19 @@ buildPythonPackage rec {
     sha256 = "14azl3gppqn90k8h4hpjilsivj92k6p1jh7mdr6p4grbww1b7sdq";
   };
 
+  patches = [
+    # Fixes a test, will be included in the next release after 0.10.2
+    (fetchpatch {
+      url = "https://github.com/geopandas/geopandas/pull/2219/commits/ac67515c9df745b672cca1669adf05eaf5cb0f3b.patch";
```

Comment:
generally we prefer not to use pull requests, as they aren't guarenteed to be stable. But these have been merged, so unlikely to be changed. And eventually should be removed

Reply:
Sounds good. The later one had 5 commits so I didn't want to add 5 patches :)